samples/matter/common/src/bridge/binding_handler.h

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+#pragma once
+
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app-common/zap-generated/ids/Commands.h>
+#include <app/CommandSender.h>
+#include <app/clusters/bindings/BindingManager.h>
+#include <controller/InvokeInteraction.h>
+#include <platform/CHIPDeviceLayer.h>
+
+#include <functional>
+
+class BindingHandler {
+public:
+	using InvokeCommand = std::function<void(chip::CommandId aCommandId, const EmberBindingTableEntry &aBinding,
```

Comment:
In the current implementation I see no reason for this not to be just a regular function pointer (it looks like it's assigned with static methods only).

Reply:
Done